### ISSUE TITLE
代码和论文有点对不上的地方

### ISSUE THREAD

**Issue by szwlh-c** [huawei-noah/Efficient-AI-Backbones#161]
你好，代码x = torch.cat([x.unsqueeze(2), x_j.unsqueeze(2)], dim=2).reshape(b, 2 * c, n, _)会对x和x''做一个拼接，然后做线性变换。但是论文中好像没有描述这一点。
![image](https://user-images.githubusercontent.com/57590287/204122145-afaec62f-41d9-4c55-91ff-69c345d5dec7.png)
请问这是为什么呢？是我哪里弄错了吗？

**Comment by iamhankai** [comment]
不好意思，论文里写错了，最新版已经更正：https://arxiv.org/pdf/2206.00272.pdf
